src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
+export type FigureColor = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: FigureColor;
+  getArea(): number;
+}
 
+function roundToHundredths(value: number): number {
+  return Math.floor(value * 100) / 100;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
 
-}
+  constructor(
+    public color: FigureColor,
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Invalid length side');
+    }
 
-export class Circle {
+    const sortedSides: number[] = [a, b, c].sort(
+      (first: number, second: number) => first - second,
```

Comment:
if you loop though the array of numbers, then you don't need to specify the types of parameters, as they can only be numbers